pep-0724.rst

```diff
@@ -50,13 +50,38 @@ might be:
 Specification
 =============
 
+Given a ``TypeGuard`` and a calling function like so:
+
+.. code-block:: python
+
+    def guard(x: P) -> TypeGuard[R]: ...
+
+    def func1(val: A):
+        if guard(val):
+            reveal_type(val) # NP
+        else:
+            reveal_type(val) # NN
+
+Each use of a ``TypeGuard`` involves five types:
+
+* P = ``TypeGuard`` parameter type
+
+* R = ``TypeGuard`` return type
+
+* A = Argument type (pre-narrowed)
+
+* NP = Narrowed type (positive)
+
+* NN = Narrowed type (negative)
```

Comment:
You don't need a blank line in-between each item of a list just to confirm -- but this might be intentional!